packages/cursorless-vscode-e2e/src/suite/performance.vscode.test.ts

```diff
@@ -11,9 +12,10 @@ import { endToEndTestSetup } from "../endToEndTestSetup";
 
 const testData = generateTestData(100);
 
-const textBasedThresholdMs = 100;
-const parseTreeThresholdMs = 500;
-const surroundingPairThresholdMs = 500;
+const smallThresholdMs = 100;
+const largeThresholdMs = 500;
```

Comment:
NTS: these are scaled to the GitHub runners that these will run on in CI, so much thresholds then you'll see locally